src/Runner.Listener/JobDispatcher.cs

```diff
@@ -256,7 +258,7 @@ private async Task EnsureDispatchFinished(WorkerDispatcher jobDispatch, bool can
                     Trace.Error(ex);
 
                     jobDispatch.WorkerCancellationTokenSource.Cancel();
-                    // make sure worker process exit before we rethrow, otherwise we might leave orphan worker process behind.
+                    // make sure worker the process exits before we rethrow, otherwise we might leave orphan worker process behind.
```

Comment:
```suggestion
                    // make sure the worker process exits before we rethrow, otherwise we might leave orphan worker process behind.
```